modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -28,15 +32,106 @@ spec:
       type: "sharedPVC"
       sharedPVC:
         claimName: "metering-nfs" <1>
-        # uncomment the lines below to provision a new PVC using the specified <2>
-        # storageClass.
+        # Uncomment the lines below to provision a new PVC using the specified storageClass. <2>
         # createPVC: true
         # storageClass: "my-nfs-storage-class"
         # size: 5Gi
 ----
 
 Select one of the configuration options below:
 
-<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany PersistentVolumeClaim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the PersistentVolume is created.
+<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany persistent volume claim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the persistent volume is created.
+
+<2> Set `storage.hive.sharedPVC.createPVC` to `true` and set the `storage.hive.sharedPVC.storageClass` to the name of a storage class with ReadWriteMany access mode. This will use dynamic volume provisioning to have a volume created automatically.
+--
+
+.  Create the following resource objects that are required to deploy an NFS server for metering. Use the `oc create -f <file-name>.yaml` command to create the object YAML files.
+..  Configure a `PersistentVolume` resource object. For example, `nfs_persistentvolume.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+  labels:
+    role: nfs-server
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteMany
+  storageClassName: nfs-server <1>
+  nfs:
+    path: "/"
+    server: REPLACEME
+  persistentVolumeReclaimPolicy: Delete
+----
+<1> Must exactly match the `[kind: StorageClass].metadata.name` field value.
+
+..  Configure a `Pod` resource object with the `nfs-server` role. For example, `nfs_server.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfs-server
+  labels:
+    role: nfs-server
+spec:
+  containers:
+    - name: nfs-server
+      image: <image_name> <1>
+      imagePullPolicy: IfNotPresent
+      ports:
+        - name: nfs
+          containerPort: 2049
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - mountPath: "/mnt/data"
+        name: local
+  volumes:
+    - name: local
+      emptyDir: {}
+----
+<1> Install an NFS server image.
```

Comment:
Hmm, we can likely get away with calling out the need to provide your own image, and we certainly shouldn't document using the NFS image that @pruan-rht using/providing to us. The only problem with this approach is that different images may expect different configurations, which can result in some of these YAML manifests we're documenting, not working. Just something I wanted to call out, but I'm not sure how much we care about that situation.

Reply:
@timflannagan @pruan-rht What do you think about, "<1> Install _your_ NFS server image." ?